content/10.methods.md

```diff
@@ -102,6 +102,30 @@ This prediction represents the probability of an honoree or author selecting a c
 
 ### Estimation of Nationality
 
+We also applied the Python package `ethnicolr` to estimate race and ethnicity, as developed by Sood and Laohaprapanon. [@arxiv:1805.02109]
+`ethnicolr` also uses United States Census data, but also adds in data curated from Wikipedia in 2009. [@doi:10.1145/1557019.1557032]
+While ethnicolr overcomes some of the limitations of wru, its international representation is still limited.
+For instance, 76% of the names in ethnicolr’s Wikipedia dataset are European in origin, and the dataset contains substantially fewer Asian, African, and Middle Eastern names. 
+
+A number of the limitations of ethnicolr are related to the source data.
+Since Wikipedia has grown substantially since 2009, we created an updated dataset and set of predictors based on the ethnicolr methodology, which are described below.
+
 #### Constructing a Name-to-Nationality Dataset
 
+To generate a training dataset for nationality prediction, we scraped Wikipedia’s category of Living People (approximately 930,000 pages).
+This category is regularly curated and allowed us to avoid pages related to non-persons and to reflect a modern naming landscape.
+For each Wikipedia page, we used two strategies to find a full birth name and nationality for that person.
+First, we pulled information from the personal details sidebar; the information in this sidebar varied widely but usually contained a full name and a place of birth.
+Second, in the body of the text of most English-language biographical Wikipedia pages, the first sentence usually begins with, for example, “John Edward Smith (born 1 January 1970) is an American novelist known for ...”
+We used regular expressions to parse out the person’s name from this structure, as well as checking that the expression after “is a” matched a list of possible nationalities.
+Using a union of these strategies, we were able to define a name and nationality for 708,493 people via their Wikipedia pages with high fidelity.
+Our Wikipedia-based process returned only a nationality or country of origin, which was more fine-grained than the broader regional patterns that we sought to examine among ISCB honorees and bioinformatics authors.
+We initially annotated based on continent, but after several iterations settled on the following categories: Hispanic (including Latin America and Iberia), African, Israeli, Muslim, SouthAsian, EastAsian, European (non-British, non-Iberian), and CelticEnglish (including United States, Canada, and Australia).
```

Comment:
Was this categorization based on any existing resources that we should mention? A citation here could be helpful to help justify the categories.